Makefile

```diff
@@ -366,6 +373,46 @@ $(COMPLETED)/uthash: vendor/uthash/.success_fetching_source $(COMPLETED)/builddi
 	touch $(COMPLETED)/uthash
 uthash: $(COMPLETED)/uthash
 
+vendor/libaio/.success_retrieving_source:
+	rm -rdf vendor/libaio/
```

Comment:
Does the `-d` flag do anything given the `-r` flag is already in use?  If not, mind changing the various `rm -rdf`'s to `rm -rf` for consistency with the rest of the file?  It's super minor but I find consistency with this kind of thing helps people new to reading a file like this who would otherwise wonder why one thing is used in one place and another is in another.

Reply:
No. `rm -d` equals `rmdir`.